updated part 1&2 to the current format and added (#1)
LumosO11y/ObservabilityOnboarding: learning/opentelemetry/part1.md

At the end of this part you should feel that you understand the necessity of OpenTelemetry
and how it helps us to collect telemetry data on our system.

## Goals
  - Understand the need of OpenTelemetry in the worlds of Observabilty.

  - Get a grasp of OpenTelemetry's abilities.

  - Undestand the diffrence between OTEL and other solutions.

## Outcome

> The outcome of this part should be a detailed markdown containing everything you have learned about OTEL and its abilities.

Reviewer: ```suggestion
The outcome of this part should be a detailed markdown containing everything you have learned about OTel and its abilities.
```

At the end of this part you should feel that you understand the necessity of OpenTelemetry
and how it helps us to collect telemetry data on our system.

## Goals

>   - understand the neeed in openTelemetry in the worlds of observabilty

Reviewer: ```suggestion
  - Understand the need of OpenTelemetry in the worlds of Observabilty.
```

>   - get a grasp of the openTelemetry abilities

Reviewer: ```suggestion
  - Get a grasp of OpenTelemetry's abilities.
```

>   - undersatnd the diffrence between OTEL and other solution 

Reviewer: ```suggestion
  - undestand the diffrence between OTel and other solutions.
```

## Outcome

> The outcome of this part should be a detailed markdown containing everything you have learned about Observability.

Reviewer: This needs to be something else because the idea isn't observability...

Author: done

In your markdown make sure there is an answer to the following questions:

1. Elaborate on the history before OpenTelemetry, how it was created and why ?

> 2. What is OpenTelemetry (aka OTEL)?

Reviewer: ```suggestion
2. What is OpenTelemetry (aka OTel)?
```

3. How does OpenTelemetry relate to Observability?
4. How did OpenTelemetry revolutionize the monitoring/observability world?
5. Jaeger vs OpenTelemetry? How do they relate?
6. Pick 3 OpenTelemetry concepts and explain them.

> 7. write 3 pro's and 3 con's of openTelemetry

Reviewer: ```suggestion
7. Write 3 pros and 3 cons of OpenTelemetry.
```

> 8. are there any other projects that are simliar to OTEL? 

Reviewer: ```suggestion
8. Are there any other projects that are similar to OTel? 
```
